presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotErrorCode.java

```diff
@@ -36,6 +37,7 @@
     PINOT_INVALID_CONFIGURATION(10, INTERNAL_ERROR),
     PINOT_DATA_FETCH_EXCEPTION(11, EXTERNAL, true),
     PINOT_REQUEST_GENERATOR_FAILURE(12, INTERNAL_ERROR), // Unable to generate Pinot request
+    PINOT_PUSH_DOWN_QUERY_NOT_PRESENT(20, USER_ERROR),
```

Comment:
Good idea to not use 13 :-P

Reply:
:) wanna start the pushdown related error code with 20+